### ISSUE TITLE
Resolve node:url error during node upgrade

### ISSUE THREAD

**Issue by CBonade** [department-of-veterans-affairs/va.gov-team#95852]
In upgrading node to version 22, we've run into an error with the `node:url` library being imported. The stack trace seems to point to jsdom and  a dependency of it "tough-cookie". This would be a good place to start investigating. We've already tried updating jsdom to the latest version, but that alone has not resolved it.

## Tasks
- [ ] Diagnose the error regarding node:url
- [ ] Implement a solution to correct the error
- [ ] Create any necessary tickets to resolve new errors that appear after that.

## Acceptance Criteria
- [ ] Node will be unblocked or we will have a new task to move forward with this effort.

**Comment by micahchiang** [comment]
Non issue because we migrated from jsdom to happydom.
